### DISCUSSION TITLE
Get Mask/Zones data for manual config (NixOS)

### DISCUSSION THREAD

**Discussion by zoechi** [blakeblackshear/frigate#13770] [Ask A Question] [answered]
In <= 0.13 The Mask/Zone editor produced a string that I could add to the Frigate config.
In >= 0.14 Frigate tries to change the config itself, but that's not possible because in NixOS the file is readonly and cannot even be made writable termporarily.
It would be great to get the option back to get the string of the drawn Mask/Zone as it should be added to the config

**Comment by NickM-27** [comment]
there is a copy button next to the mask for this purpose
<img width="354" alt="Screenshot 2024-09-16 at 9 58 46 AM" src="https://github.com/user-attachments/assets/6351496b-a2f9-4ec7-9daf-71c83e275e75">

**Reply by zoechi** [reply]
Thanks a lot. I missed that one

**Reply by zoechi** [reply]
I don't see this in 0.14.1
![Screenshot_20240917_072156](https://github.com/user-attachments/assets/af38b1c4-c612-4ae9-be56-cad8ced1e3dd)

**Reply by NickM-27** [reply, marked as answer]
You have to save the mask first, it'll still save even if it can't overwrite the config

**Reply by zoechi** [reply]
Ok, that was it.
